test/http-request.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpRequest } from '../src/http-request';
+import nock from 'nock';
+
+describe.only('HttpRequest', () => {
```

Comment:
should this .only still be here?

Reply:
Ouch, good catch! 👍 ... I should probably add an ESLint plugin for Jest to prevent this.